### ISSUE TITLE
GIFs exported by Kap do not loop indefinitely by default

### ISSUE THREAD

**Issue by frozenpandaman** [wulkano/Kap#940]
**macOS version:** 10.14.6
**Kap version:** 3.3.2, installed via Homebrew cask

Exporting GIFs using Kap results in files that play once, then stop (until the page/image is reloaded). I'd assume the default behavior for these should instead be to loop forever, as normal GIFs do.

**Comment by Thatkookooguy** [comment]
I would suggest having an option to either loop or not. I use KAP to capture animated gifs for slideshows and I don't want them to play as a loop while I'm lecturing.

**Comment by sindresorhus** [comment]
I agree. We should loop indefinitely by default and have a checkbox in the editor to turn off looping.

**Comment by karaggeorge** [comment]
Hmm, I think we have looping on by default in the settings: https://github.com/wulkano/Kap/blob/main/main/common/settings.ts#L65-L68

Maybe we can expose that toggle in the editor too

**Comment by skllcrn** [comment]
Seems that this issue can be closed, as looping is on by default, which is what you would expect. This can be toggled off in preferences as most people do not change it.
